Library/Tracking/KSRAnalyticsTests.swift

```diff
@@ -640,7 +640,13 @@ final class KSRAnalyticsTests: TestCase {
     let project = Project.cosmicSurgery
     let reward = Reward.template
 
-    ksrAnalytics.trackAddOnsContinueButtonClicked(project: project, reward: reward, refTag: .recommended)
+    ksrAnalytics
```

Comment:
After you remove the pledgeProperties, this method will fail. I would replace this tracking method with another one that is still using pledge properties so we can keep the coverage up.

Reply:
Do you mean another test method or another tracking method inside `KSRAnalytics`? If I add pledge properties wouldn't that still be sending that surplus data for that event?